drivers/win7/driver.c

```diff
@@ -872,8 +953,8 @@ DriverDeviceControl(
             // --
             // -- read MSR to check if it was written
             // --
-//            _rdmsr( _msr_addr, &_eax, &_edx );
-//            DbgPrint( "[chipsec][IOCTL_WRMSR] RDMSR( 0x%x ) --> 0x%08x%08x\n", _msr_addr, _edx, _eax );
+          //  _rdmsr( _msr_addr, &_eax, &_edx );
+          //  DbgPrint( "[chipsec][IOCTL_WRMSR] RDMSR( 0x%x ) --> 0x%08x%08x\n", _msr_addr, _edx, _eax );

```

Comment:
Why are we maintaining dead code?  I would have been fine with just a removal or no change at all.